Missing npm dependency

When running `npm run  build` you got the following error:

```
root@95a52cdbe85a:/tyra-master/frontend# npm install                                                                                     
npm WARN sass-loader@6.0.6 requires a peer of webpack@^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0 but none was installed.
npm WARN webpack-dev-server@2.7.1 requires a peer of webpack@^2.2.0 || ^3.0.0 but none was installed.
npm WARN tyra@0.0.0 license should be a valid SPDX license expression
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 114 packages in 7.763s
root@95a52cdbe85a:/tyra-master/frontend# npm run build                                                                                   

> tyra@0.0.0 build /tyra-master/frontend
> webpack --progress --colors

Hash: 7844a6039f9ad1830e9b
Version: webpack 1.15.0
Time: 5260ms
    Asset     Size  Chunks             Chunk Names
output.js  4.34 MB       0  [emitted]  main
    + 788 hidden modules

ERROR in ./components/rankcorr-chart/component.js
Module not found: Error: Cannot resolve module 'react-faux-dom' in /tyra-master/frontend/components/rankcorr-chart
 @ ./components/rankcorr-chart/component.js 15:20-45
```

You can fix that installing that package:

```
root@95a52cdbe85a:/tyra-master/frontend# npm install react-faux-dom
```

But it should be added to the proper place in code

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -30,14 +30,15 @@
     "react": "15.6.1",
     "react-datepicker": "~0.39.0",
     "react-dom": "15.6.1",
+    "react-faux-dom": "^4.0.3",
     "react-highcharts": "^11.5.0",
     "react-nvd3": "^0.5.7",
+    "react-redux": "^4.3.0",
     "react-table": "^3.1.4",
     "react-tabs": "~0.8.2",
-    "react-redux": "^4.3.0",
-    "redux":"^3.0.0",
     "react-vis": "^1.6.7",
-    "reactable": "~0.14.1"
+    "reactable": "~0.14.1",
+    "redux": "^3.0.0"
   },
   "devDependencies": {
     "babel-cli": "^6.5.2",

